src/malloc_wrapper.cc

```diff
@@ -46,7 +46,7 @@ void gnu_backtrace()
 	}
 
 	for (j = 0; j < nptrs; j++)
```

Comment:
[llvmlibc-callee-namespace]: '__real_malloc' must resolve to a function declared within the '__llvm_libc' namespace